freqtrade/optimize/analysis/recursive_helpers.py

```diff
@@ -101,6 +103,5 @@ def start(config: Config):
                 RecursiveAnalysis_instances)
         else:
             logger.error("There were no strategies specified neither through "
-                         "--strategy nor through "
-                         "--strategy-list "
+                         "--strategy "
```

Comment:
You might want to update the first line, too, to be singular.

it's now kinda odd, and reads like there's something missing.